### PR TITLE
Update rubygem-fog-proxmox to 0.15.0

### PR DESCRIPTION
**Pull request by ekohl**
I opened https://github.com/fog/fog-proxmox/pull/83 to remove the files we `%exclude`.

### CODE REVIEW

File: packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec [hunk 1]
```diff
@@ -1,115 +1,89 @@
...
-%exclude %{gem_instdir}/Gemfile.lock
+%doc %{gem_instdir}/CHANGELOG.md
+%exclude %{gem_instdir}/Gemfile
 %doc %{gem_instdir}/README.md
-%{gem_instdir}/Rakefile
+%exclude %{gem_instdir}/Rakefile
 %doc %{gem_instdir}/docs
-%doc %{gem_instdir}/examples
-%{gem_instdir}/fog-proxmox.gemspec
+%{gem_instdir}/examples
```
Comment (dgoetz) [inline packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec:79]: examples was flagged as doc before, is this change intended?

Comment (ekohl) [reply packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec:79]: No, good catch. Updating.
